alerts/jvm/heap-memory-usage.json

```diff
@@ -0,0 +1,26 @@
+{
+    "displayName": "JVM - Heap Memory Usage Near Max",
+    "documentation": {
+        "content": "Alerts on JVM heap memory usage being near maximum. When the JVM reaches its max, depending on JVM configuration could lead to Out of Memory errors.",
```

Comment:
Minor edit: 

"Alert for JVM heap memory usage being near maximum. When the JVM reaches its max, depending on the JVM configuration, this could lead to Out of Memory errors.",